### DISCUSSION TITLE
[ADFS] Disable login screen an redirect to SAML provider

### DISCUSSION THREAD

**Discussion by medamines1** [keycloak/keycloak#17036] [Q&A] [answered]
I have added a login to my Backoffice using adfs (SAML) so now I have this screen when I want to log in:

![image](https://user-images.githubusercontent.com/13109656/218468167-ce491123-58b6-4a68-95b9-b4b5ae5763d0.png)

I want to stop showing this screen and redirect to the adfs directly.

The only possible solution I thought of was to change the template in keycloak to redirect me automatically, but I don't think this is the best approach.

Any help?

**Comment by bonnm** [comment]
https://www.keycloak.org/docs/latest/server_admin/index.html#default_identity_provider

**Comment by medamines1** [comment]
Thank you @bonnm for your answer but I get this error whenever I click  on the gear icon: 

![image](https://user-images.githubusercontent.com/13109656/218971372-b190b8d8-2449-4750-b09d-ed0844e3c9f0.png)

**Comment by bonnm** [comment, marked as answer]
Seems to be a bug of the new admin console. You can try to start keycloak with the old console enabled to be able to change the flow settings.

**Reply by medamines1** [reply]
Thank you for your time.
